scripts/lint/lint-fix-changes.sh

```diff
@@ -23,22 +23,22 @@ if [[ -e "${SWIFT_LINT}" ]]; then
     ##### Fix files or exit if no files found for fixing #####
     if [ "$count" -ne 0 ]; then
         echo "Found files to fix! Running swiftLint --fix..."
-        
+
         # Run SwiftLint --fix on each file
         for ((i = 0; i < count; i++)); do
             file_var="SCRIPT_INPUT_FILE_$i"
             file_path=${!file_var}
             echo "Fixing $file_path"
-            $SWIFT_LINT --fix --path "$file_path"
+            $SWIFT_LINT --fix "$file_path"
```

Comment:
`--path` was removed in a recent versions of swiftlint. This should be compatible with older versions of swiftlint since it's been deprecated for at least 2 years before that.